Documentation/Blazorise.Docs/Components/Commercial/PlanSwitch.razor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using Blazorise.Docs.Services;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Docs.Components.Commercial;
+
+public partial class PlanSwitch : IDisposable
+{
+    private Background GetSelectedBackground( string value ) => SelectedValue == value ? Background.Primary : Background.Default;
```

Comment:
I don't understand these getters, if the **SelectedValue** is already equal to the value that's in the argument it chooses Background.Primary, otherwise Default?

Reply:
It just applies the primary color to the selected button. That's it.

Reply:
I still don't understand this at all personally... But as long as it's working as expected, I'm ok with it, then.